test/integration/job/job_test.go

```diff
@@ -1681,24 +1682,29 @@ func TestJobPodReplacementPolicy(t *testing.T) {
 	}
 	cases := map[string]struct {
 		podReplacementPolicyEnabled bool
+		deletePods                  bool
```

Comment:
I doesn't look like there is a test that has this as false. Let's not add a parameter if we don't need it.

Reply:
the 2 test cases for Failed policy use `failPods` instead of `deletePods`

Reply:
it is false in the following test cases:
* https://github.com/kubernetes/kubernetes/pull/121274/files#diff-ace35385ec6f5400e8ccbfe8e9f35c2f9573d0fc05889071dc9c4b36e5e975f7R1763
* https://github.com/kubernetes/kubernetes/pull/121274/files#diff-ace35385ec6f5400e8ccbfe8e9f35c2f9573d0fc05889071dc9c4b36e5e975f7R1763

Reply:
ah, missed them, thanks